### DISCUSSION TITLE
How can I use ERP file instead of EOP file in groops?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#170] [Q&A]
Hello developers! I'd like to use set of GNSS precise products from a certain IGS analysis center. However, IGS analysis center such as WHU does not provide EOP file, but ERP file. In this case, how can I deploy a PPP program using ERP file? Also，when EOP updates, is it necessary to use EOP file downloaded from TUG FTP? Is there any other way to update EOP file by user(with groops interface) ?Could you please give me some instructions about it?Many thanks!

**Comment by tmayerguerr** [comment]
Hi,

At the moment GROOPS is not able to read IGS ERP files. Maybe you can convert the files to the standard IERS file formats? In this case you can use the programs [IersC04IAU2000EarthOrientationParameter](https://groops-devs.github.io/groops/html/IersC04IAU2000EarthOrientationParameter.html) or [IersRapidIAU2000EarthOrientationParameter](https://groops-devs.github.io/groops/html/IersRapidIAU2000EarthOrientationParameter.html) (To be used in [earthRotation](https://groops-devs.github.io/groops/html/earthRotationType.html)->IERS2010 (or IERS2010b).

regards
Torsten
